package.json

```diff
@@ -10,7 +10,7 @@
     "publish": "node ./scripts/publish.js",
     "publish:private": "node ./scripts/publish-private.js",
     "release": "node ./scripts/release.js",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
Can we restrict this to just the puppeteer tests in our Jest config? That way our unit tests can still run in parallel.

Reply:
as far as i can tell, there's no config file options for "runInBand" 😕